config/crd/bases/kueue.x-k8s.io_capacities.yaml

```diff
@@ -82,6 +82,15 @@ spec:
                   vice versa. \n The name style is similar to label keys. These are
                   just names to link QCs together, and they are meaningless otherwise."
                 type: string
+              queueingStrategy:
+                default: StrictFIFO
+                description: 'QueueingStrategy indicates the queueing strategy of
+                  the workloads across the queues in this Capacity. This field is
+                  immutable. Current Supported Strategies: - StrictFIFO: workloads
```

Comment:
you need an extra empty line in the gocomment :)

Reply:
(⊙o⊙)… Added